generators/ci-cd/templates/github-ci.yml.ejs

```diff
@@ -0,0 +1,69 @@
+<%#
+ Copyright 2013-2019 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+name: Application CI
+on: [push, pull_request]
+jobs:
+    applications:
+        name: <%= baseName %> test suite
+        runs-on: ${{ matrix.os }}
+        timeout-minutes: 40
+        strategy:
+            fail-fast: false
+            matrix:
+                node_version: [<%= NODE_VERSION %>]
+                os: [ubuntu-latest]
+                java_version: ['8.x', '11.x']
```

Comment:
I'd prefer to keep only JDK 11, rather than support both version, as we'll probably remove JDK 8 soon

Reply:
I have no hard preference here, however, I think it would be good to support until `v7`.